index.html

```diff
@@ -56,6 +56,96 @@ <h3><a href="#">physionic</a></h3>
         </div>
     </header>
     <!-- ========== End hero ========== -->
+    <footer>
```

Comment:
![image](https://user-images.githubusercontent.com/44810632/154468889-430a15c3-3cc9-4ff6-9221-c37424f75896.png)
review design, it does not match